pandas/tseries/tests/test_period.py

```diff
@@ -2984,6 +2984,12 @@ def test_to_datetime_1703(self):
         result = index.to_datetime()
         self.assertEqual(result[0], Timestamp('1/1/2012'))
 
+    def test_to_datetime_dimensions(self):
+        df = DataFrame({'a': ['1/1/2012', '1/2/2012'],
+                        'b': ['12/30/2012', '12/31/2012']})
+        with tm.assertRaisesRegexp(ValueError, "dimensions"):
```

Comment:
so we need to normalize these messages; I like what you did for ``to_numeric``, essentially repeating the doc-string